app/src/main/java/com/cs6018/canvasexample/DrawingInfoViewModel.kt

```diff
@@ -32,12 +32,16 @@ class DrawingInfoViewModel(private val repository: DrawingInfoRepository) : View
         repository.addNewDrawingInfo(drawingInfo)
     }
 
+    private fun updateDrawingInfoTitle(title:String) {
+        repository.updateDrawingInfoTitle(title, activeDrawingInfo.value?.id ?: 0)
+    }
+
     fun setActiveCapturedImage(imageBitmap: Bitmap?) {
         (activeCapturedImage as MutableLiveData).value = imageBitmap
         Log.d("DrawingInfoViewModel", "Bitmap is set as activeCapturedImage.")
     }
 
-    fun addDrawingInfoWithRecentCapturedImage(context: Context): String? {
+    fun addDrawingInfoWithRecentCapturedImage(context: Context, drawingTitle: String): String? {
```

Comment:
- update function name ( not only image but also title )
- fix failed test `testAddDrawingInfoWithRecentCapturedImage`